src/desktop/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "trinity-desktop",
   "productName": "Trinity",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
In the future, this can automatically be bumped by running `yarn prepare-release` and will also create a git tag with the correct prefix (`desktop-v`)